### ISSUE TITLE
How to stop autoscroll in command line

### ISSUE THREAD

**Issue by TheRogue27** [termux/termux-app#684]
I am having an issue with this. When the output keeps flowing i cant scroll up as i get returned to the current output. Is there any fix for this?

**Comment by xeyownt** [comment]
I second that.

Currently if the terminal runs an application that outputs regularly some text, the scroll position is automatically reset to the bottom. There should be an option to lock to the current position. In my opinion this should even be the default option (scroll position always locked except if terminal shows the last line).

**Comment by Grimler91** [comment]
I agree that some way to freeze the terminal would be useful (like ctrl + s on normal gnu/linux systems). 

A workaround is to use tmux, that gives you the possibility to freeze the terminal and scroll around.

**Comment by xeyownt** [comment]
True.
tmux works for me, so I guess this feature is more a nice-to-have now.

**Comment by buffermet** [comment]
I can't believe this hasn't been implemented yet. It's one of the most basic features of a terminal.

**Comment by maxried** [comment]
It's more of a usability faux pas for a terminal application...

**Comment by ioogithub** [comment]
Please revisit this, I can't work with my current process due to constantly fighting with termux that is insisting to stealing the focus.

**Comment by buffermet** [comment]
> Long tap until the popup menu shows. Then scroll up without loosing touch with the screen and without touching the popup menu.

That seems to do the trick, thank you!

**Comment by Grimler91** [comment]
This feature was added in commit https://github.com/termux/termux-app/commit/5fc2b4cd4ae59a7cbfccea60c76019e96efb847c, through the `⇳` button in extra-keys , and is part of the 0.119.0 beta releases (but not the 0.118.3 release)
